ext/zend_test/test.c

```diff
@@ -884,11 +884,92 @@ static void le_throwing_resource_dtor(zend_resource *rsrc)
 	zend_throw_exception(NULL, "Throwing resource destructor called", 0);
 }
 
+// Returns a newly allocated DNF type `Iterator|(Traversable&Countable)`.
+//
+// We need to generate it "manually" because gen_stubs.php does not support codegen for DNF types ATM.
+static zend_type create_test_dnf_type(void) {
+	zend_string *class_Iterator = zend_string_init_interned("Iterator", sizeof("Iterator") - 1, true);
+	zend_alloc_ce_cache(class_Iterator);
+	zend_string *class_Traversable = zend_string_init_interned("Traversable", sizeof("Traversable") - 1, true);
+	zend_alloc_ce_cache(class_Traversable);
```

Comment:
```suggestion
	zend_string *class_Traversable = ZSTR_KNOWN(ZEND_STR_TRAVERSABLE);
```
Maybe? Also helps to check that a known string doesn't cause issues for extensions.